### DISCUSSION TITLE
case insensitive search for tags

### DISCUSSION THREAD

**Discussion by tingan** [spatie/laravel-tags#299] [Ideas]
hi
,

Tag::where('name', 'LIKE', '%'.  $request->get( 'tag' ) . '%')->orWhere('type', 'LIKE', '%'.  $request->get( 'tag' ) . '%')
`
I found that the type field can search by case insensitive, but the name field is case sensitive, how to make name field also case insensitive? thx.

**Comment by cdbeaton** [comment]
I've just come across the same issue. It seems very strange to me that tag names are case sensitive. Can anyone explain why this should be the case, or should this be changed in a future update?

**Edit:** On further testing, it appears that tags with the same name but different capitalisation have the exact same slug, which definitely can't be the intended behaviour as slugs should always be unique. This is beginning to look like a bug.
